tests/cloudpickle_test.py

```diff
@@ -440,6 +442,48 @@ def method(self, x):
         mod1, mod2 = pickle_depickle([mod, mod])
         self.assertEqual(id(mod1), id(mod2))
 
+    def test_dynamic_modules_cache(self):
```

Comment:
This is not a cache. A cache has specific semantics which are not the semantics of this hold. It's more like a list of reference to the known active dynamic modules.